docs/dg/dev/troubleshooting/troubleshooting-performance-issues/redis-session-lock-excpetions.md

```diff
@@ -67,3 +88,39 @@ Explore [architecture performance guidelines](/docs/dg/dev/guidelines/performanc
 
 ### Leverage APIs
 Headless scenarios aren't usually impacted by Session Locking challenges. Evaluate if you can adjust calls that would normally target Yves to target APIs instead.
+
+### Do not use locks
+
+```php
+// config/Shared/config_default.php
+$config[SessionConstants::YVES_SESSION_SAVE_HANDLER] = SessionRedisConfig::SESSION_HANDLER_REDIS;
+```
+This approach optimistically assumes that there will be no concurrent requests from the client, which could otherwise cause session data inconsistencies. 
+For instance, this could occur if products are added to the cart simultaneously from multiple browser tabs.
+
```

Comment:
Can you please add here as a "warning" message: 

Disabling session locking is possible but should only be done after thoroughly assessing the risks and implementing necessary safeguards. For optimal performance, data integrity, and customer experience, we strongly recommend keeping session locks enabled and focusing on optimizing lock management and application performance.

Reply:
added